app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/adresslookup/Address.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.model.adresslookup
+
+import play.api.libs.json._
+
+case class Country(code: String, name: Option[String])
+
+case class Address(lines: Seq[String], postcode: Option[String], country: Country)
```

Comment:
Interesting - I'd thought this would return UK addresses only. 

Not sure if customs agents will all be UK based.

One to talk about :-)